PlayerSkeleton.java

```diff
@@ -8,6 +10,7 @@ public int pickMove(State s, int[][] legalMoves) {
 	public int pickMove(State s, int[][] legalMoves, double[] weights) {
 		//Test all possible moves of the piece on the board and choose the move with the best heuristic score 
 		int best = tryPossibleMoves(s, legalMoves, weights);
+		System.out.println("Picked best move for " + Arrays.toString(weights) + " with score of [" + best + "]");
```

Comment:
`best `is not best score, `best` is an int indicating which move is the best.

Reply:
ah ok actually i just wanted to add a print statement so i know it's running HAHA but i can just remove that